Dockerfile

```diff
@@ -1,18 +1,9 @@
-FROM maven:3.6.3-ibmjava-8-alpine AS MAVEN_BUILD_ENVIRONMENT
-
-COPY pom.xml /tmp/
-COPY src /tmp/src/
-WORKDIR /tmp/
-
-RUN mvn clean package --no-transfer-progress
-
-FROM openjdk:8-jre
+FROM openjdk:15-slim
 
 ENV TZ=Europe/Oslo
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
 VOLUME /tmp
-COPY --from=MAVEN_BUILD_ENVIRONMENT /tmp/target/fdk-concept-harvester.jar app.jar
+COPY /target/fdk-concept-harvester.jar app.jar
```

Comment:
det går an å styre jar navn i pom.xml til å være app.jar, så slipper du å rename her

Reply:
Ja, men må jo fortsatt kopiere, dvs `COPY /target/app.jar app.jar`
Om vi virkelig ikke vil rename så er jeg mer for at vi beholder nåværende navn
```
COPY /target/fdk-concept-harvester.jar fdk-concept-harvester.jar
CMD java -jar $JAVA_OPTS fdk-concept-harvester.jar
```